R/paginate_listing.R

```diff
@@ -35,12 +29,9 @@
 #' paginate_listing(lsting, cpp = 100, lpp = 40)
 #'
 #' paginate_listing(lsting, cpp = 80, lpp = 40, verbose = TRUE)
-#' @export
 #'
-#' @return for `paginate_listing` a list containing separate
-#' `listing_df` objects for each page, for `pag_listing_indices`,
-#' a list of indices in the direction being paginated corresponding
-#' to the individual pages in that dimension.
+#' @export
+#' @rdname paginate
 paginate_listing <- function(lsting,
```

Comment:
paginate_listings and tables are deprecated functions that add more confusion than anything. I think it happened during migration from rtables/rlistings to formatters of exporters. We should start deprecation cycle